dotnet/src/Connectors/Connectors.Memory.MongoDB/README.md

```diff
@@ -0,0 +1,46 @@
+﻿# Microsoft.SemanticKernel.Connectors.Memory.MongoDB
```

Comment:
I think we should also add in the README how to configure the cluster to pass our tests as well. That may be useful for folks.